bar/90_contents/Engine/setAccessInfo.js

```diff
@@ -253,79 +189,43 @@ function(request) {
             }
           } else if (setInfo.srcType == "Google") {
             if (setInfo.srcType == accessInfo[i].srcType && setInfo.srcAccountName == accessInfo[i].srcAccountName && setInfo.refreshToken != accessInfo[i].refreshToken) {
               try {
                 //TODO:check if the refreshToken is correct.
               } catch (e) {
-                return {
-                  status : 400,
-                  headers : {"Content-Type":"application/json"},
-                  body: ['{"error": "Required paramter is not access Google server."}']
-                };
+                return createResponse(400, {"error": "Required paramter is not access Google server."})
               }
               accessInfo[i].refreshToken = setInfo.refreshToken;
               personalBoxAccessor.put(pathDavName, "application/json", JSON.stringify(accessInfo));
             } else if (setInfo.srcType == accessInfo[i].srcType && setInfo.srcAccountName == accessInfo[i].srcAccountName && setInfo.refreshToken == accessInfo[i].refreshToken) {
-              return {
-                status : 400,
-                headers : {"Content-Type":"application/json"},
-                body: ['{"error": "Required paramter set is not change."}']
-              };
+              return createResponse(400, {"error": "Required paramter set is change."})
             } else {
-              return {
-                status : 400,
-                headers : {"Content-Type":"application/json"},
-                body: ['{"error": "Required paramter set is incorrect."}']
-              };
+              return createResponse(400, {"error": "Required paramter set is incorrect."})
             }
           } else {
             if (setInfo.srcType == accessInfo[i].srcType && setInfo.srcAccountName == accessInfo[i].srcAccountName && setInfo.pw == accessInfo[i].pw && setInfo.srcUrl == accessInfo[i].srcUrl) {
-              return {
-                status : 400,
-                headers : {"Content-Type":"application/json"},
-                body: ['{"error": "Required paramter set is not change."}']
-              };
+              return createResponse(400, {"error": "Required paramter set is not change."})
             //} else if () {
             } else { // e.g. Google
               // srcType is not EWS.
               // not supported now!
-              return {
-                status : 400,
-                headers : {"Content-Type":"application/json"},
-                body: ['{"error": "Required srcType is not supported."}']
-              };
+              return createResponse(400, {"error": "Required srcType is not supported."})
             }
           }
         }
       } catch (e) {
-        return {
-          status : e.code,
-          headers : {"Content-Type":"application/json"},
-          body: [JSON.stringify({"code": e.code, "message": e.message})]
-        };
+        return createResponse(e.code, {"message": e.message})
```

Comment:
1. I think e.code is not integer. Please check.  
1. Please return in the same format. It is hard to handle when sometimes the key is "error", and another time it is "message".